src/v/raft/consensus.h

```diff
@@ -449,7 +449,7 @@ class consensus {
     };
 
     // precondition: is_elected_leader() must be true.
-    suppress_heartbeats_guard suppress_heartbeats(vnode);
+    inflight_appends_guard append_inflight(vnode);
```

Comment:
nit: would be nice to have a verb in the function name, something like "track_inflight_append", with the current name there is an impression that we are appending something